CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Unreleased Changes
+
+## vX.X
+
+Month, DD, YYYY
+
+### BREAKING CHANGES
+
+- [go package] (Link to PR) Description @username
+
+### FEATURES
+
+- [go package] (Link to PR) Description @username
+
+### IMPROVEMENTS
+
+- [go package] (Link to PR) Description @username
+
+### BUG FIXES
+
+- [go package] (Link to PR) Description @username
```

Comment:
Just to clarify: the workflow here will be to populate the CHANGELOG-PENDING.md on each PR. Then, before a release gets cut, the content of the CHANGELOG-PENDING.md gets copied over (or rather cut & pasted) to the CHANGELOG.md. 

Can we cut a post-devnet release right after this here gets merged? After https://github.com/celestiaorg/celestia-node/issues/226 we can do a small patch release. (cc @renaynay @Wondertan).

Reply:
Correct, I'm planning to have a similar action to ensure the CHANGELOG.md is updated during manual releases. The manual release process is TBD.